src/server/actions/getUserSettings.ts

```
@@ -86,7 +85,7 @@ export async function getUserSettings(userId: string) {
             isHidden: space.quoteIsHidden,
           },
           background: space.background ?? undefined,
-          icon: defaultSettings.spaces[0]!.icon, // Fallback to first default space icon
+          icon: defaultSettings.spaces[0]!.icon,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)